app/helpers/application_helper.rb

```diff
@@ -34,6 +34,13 @@ def date_string(start_date, end_date)
     result
   end
 
+  # Returns the time with the conference's timezone
+  def set_offset(time, timezone)
+    time = time.in_time_zone(timezone)
+    offset = time.utc_offset
+    time + -offset
```

Comment:
adding a negative value is the same as subtracting the value itself!!!!

`time - offset`